### PR TITLE
Set input text length to 16

### PR DESCRIPTION
**Pull request by teyim**
# Description 📣

Reduce input text length to 16 characters

## Type ✨

- [x] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Documentation

# Tests 🛠️
Manual test

```sh
# Here's some code block to paste some code snippets
```

### CODE REVIEW

**Comment by rohan-chaturvedi** [comment]
@teyim is this ready for review?

**Comment by teyim** [comment]
yes @rohan-chaturvedi

File: frontend/components/apps/NewAppDialog.tsx [hunk 1]
```diff
@@ -483,7 +483,7 @@ export default function NewAppDialog(props: { appCount: number; organisation: Or
                               id="appname"
                               className="text-lg"
                               required
-                              maxLength={64}
+                              maxLength={16}
```
Comment (rohan-chaturvedi) [inline frontend/components/apps/NewAppDialog.tsx]:
```suggestion
                              maxLength={32}
```

File: frontend/components/onboarding/TeamName.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ export const TeamName = (props: TeamNameProps) => {
         id="teamname"
         className="text-lg"
         required
-        maxLength={64}
+        maxLength={16}
```
Comment (rohan-chaturvedi) [inline frontend/components/onboarding/TeamName.tsx]:
```suggestion
        maxLength={32}
```

**Review by rohan-chaturvedi — CHANGES_REQUESTED** [review]
This looks good! Lets just make the max length 32, to allow for more descriptive names.

File: frontend/components/apps/NewAppDialog.tsx (hunk 1 above)
Comment (teyim) [reply frontend/components/apps/NewAppDialog.tsx]: ok, i will update that

**Comment by teyim** [comment]
Hi @rohan-chaturvedi , i have modified this by creating a global constant to hold the max input string length..i think this approach is better.. please review

File: frontend/components/apps/NewAppDialog.tsx [hunk 3]
```diff
@@ -32,6 +32,7 @@ import {
   getUserKxPrivateKey,
   getUserKxPublicKey,
 } from '@/utils/crypto'
+import { MAX_INPUT_STRING_LENGTH, MAX_STRING_LENGTH } from '@/constants'
```
Comment (rohan-chaturvedi) [inline frontend/components/apps/NewAppDialog.tsx]:
```suggestion
import { MAX_INPUT_STRING_LENGTH } from '@/constants'
```

I think only one import is required here

File: frontend/constants/index.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+// max length for an input string
+export const MAX_INPUT_STRING_LENGTH=32
```
Comment (rohan-chaturvedi) [inline frontend/constants/index.ts]:
```suggestion
export const MAX_INPUT_STRING_LENGTH=32

```

Adding a newline at EOF

**Review by rohan-chaturvedi — CHANGES_REQUESTED** [review]
@teyim Thanks for this. Not 100% sure this solution will work for all use cases, as input field max-length can vary depending on the context, but lets try this solution and see how much we can standardize it.

Just requesting a couple of fixes. Lets go ahead with this.

**Review by rohan-chaturvedi — APPROVED** [review]

**Pull request merged**